schema/2.0.0/seurat_encoding.md

```diff
@@ -237,4 +237,64 @@ The matrix with reductions is stored in the `cell.embeddings` slot of `SeuratObj
 
 ### Dataset Metadata
 
-At the moment a Seurat object downloaded from cellxgene does not contain the dataset metadata stored in the AnnData slot `uns`.
+The dataset metadata from the AnnData slot `uns` is stored in the Seurat object in the slot `misc`. 
+
+<table><tbody>
+    <tr>
+      <th>Access</th>
+      <td><code>seurat_object@misc</code></td>
+    </tr>
+    <tr>
+      <th>Value</th>
+        <td><code>list[character|list]</code></td>
+    </tr>
+</tbody></table>
+<br>
+
+Only fields defined in the [schema](./schema.md/#uns-dataset-metadata) are transferred from the AnnData to the Seurat object:
+
+
+<table><tbody>
+    <tr>
+      <th>Name</th>
+      <th>Value</th>
+      <th>Optional</th>
+    </tr>
+    <tr>
+      <td>schema_version</td>
+      <td><code>character</code></td>
+      <td>No</td>
+    </tr>
+    <tr>
+      <td>title</td>
+      <td><code>character</code></td>
+      <td>No</td>
+    </tr>
+    <tr>
+      <td>X_normalization</td>
+      <td><code>character</code></td>
+      <td>No</td>
+    </tr>
+    <tr>
+      <td>feature_name</td>
```

Comment:
Just notice that `feature_name` isn't listed in:
* https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/2.0.0/schema.md#uns-dataset-metadata

Should it be?

Reply:
Thanks for catching this, it's an error